src/groups/bmq/bmqeval/bmqeval_simpleevaluator.t.cpp

```diff
@@ -287,7 +314,7 @@ static void test3_evaluation()
     struct TestParameters {
         const char* expression;
         bool        expected;
-        bool        expectCompilationFailure;
+        bool        expectCompilationFailure = false;
```

Comment:
This is a modern C++ feature

Reply:
Moved compilation failures to its own test